src/components/Criteria/CriteriaCardSelect.tsx

```diff
@@ -0,0 +1,144 @@
+import { Select, SelectOptionType } from '@folio/stripes/components';
+import React, { useMemo } from 'react';
+import { Field } from 'react-final-form';
+import {
+  CriteriaGroupType,
+  CriteriaTerminalType,
+} from '../../types/CriteriaTypes';
+import { useIntl } from 'react-intl';
+
+export default function CriteriaCardSelect({
+  name,
+  root = false,
+  patronOnly = false,
+}: {
+  name: string;
+  root?: boolean;
+  patronOnly?: boolean;
+}) {
+  const selectDefaultValue = useMemo(() => {
+    if (root) {
+      return CriteriaTerminalType.PASS;
+    } else {
+      return CriteriaGroupType.ALL_OF;
+    }
+  }, [root]);
+
+  const intl = useIntl();
+
+  const selectOptions = useMemo(() => {
+    const options: SelectOptionType<
+      CriteriaGroupType | CriteriaTerminalType
+    >[] = [
```

Comment:
look how huge component looks just because of this array, why not to move it to separate file like [{labelId, value}, ...} and translate it in component `options.map()`

Reply:
or you can create a function and pass intl there, there are many variants